doc/devel/contribute.rst

```diff
@@ -183,6 +183,23 @@ please cite us following the :doc:`/project/citing` guidelines.
 If you have developed an extension to Matplotlib, please consider adding it to our
 `third party package <https://github.com/matplotlib/mpl-third-party>`_  list.
 
+
+.. _generative_ai:
+
+
+
+Use of Generative AI
+--------------------
+
+We expect authentic engagement in our community.  Do not post verbatim output
+of Large Language Models or similar generative AI as comments on GitHub or our
+discourse server as your own work.  For proposed code changes ensure that you
```

Comment:
Is that sufficient? The wording allows to cop&paste AI code verbatim as long as it’s reasonable and the user can explain it. Do we know the legal/copyright implications of that?

Reply:
I pushed some slight rewording, not sure if I made it better or worse though.